json/contestants.json

```diff
@@ -10,6 +10,16 @@
 			"twitter": "",
 			"referral": ""
 		},
+			{
+   			"name" : "Atul Kumar Agrawal",
+			"imageurl" : "contestant/atul.jpg",
+			"about" : "Coder.Geek",
+			"college" : "Veer Surendra Sai University Of Technology,Burla,Sambalpur",
+			"facebook" : "https://www.facebook.com/msatul1305",
+			"github" : "https://github.com/msatul1305",
+			"twitter" : "https://twitter.com/ms_atul",
+			"referral" : ""
+}
```

Comment:
Put a comma here. Also do not open a new PR for adding image separately. It can be done by adding image to same branch from which this PR is made.

Reply:
done (y)